src/hipercow/cli.py

```diff
@@ -26,6 +29,43 @@
 )
 from hipercow.task_create import task_create_shell
 from hipercow.task_eval import task_eval
+from hipercow.util import truthy_envvar
+
+# This is how the 'rich' docs drive things:
```

Comment:
This "rich" ambiguity is going to get quite confusing...

Reply:
yeah, it's quite odd